src/core.jl

```diff
@@ -385,22 +365,6 @@ ne(g) / (nv(g) * (nv(g) - 1))
 @traitfn density(g::::(!IsDirected)) =
 (2 * ne(g)) / (nv(g) * (nv(g) - 1))
 
-
-"""
-    squash(g)
-
```

Comment:
`squash` isn't general enough to be part of the core, so we moved it into SimpleGraphs.

Reply:
I would keep squast in core and add a not-implement exception as it is a very handy function and it would be useful if a lot of graphs implement it.

Reply:
OK. I can do that.